home_workshop_misc/init.lua

```diff
@@ -79,60 +79,13 @@ minetest.register_node("home_workshop_misc:beer_mug", {
 	sounds = default.node_sound_glass_defaults(),
 	selection_box = beer_cbox,
 	on_use = function(itemstack, user, pointed_thing)
-		if not minetest.is_creative_enabled_for(user:get_player_name()) then
+		if not minetest.is_creative_enabled(user:get_player_name()) then
```

Comment:
`minetest.is_creative_enabled` is 5.3.0+ only. (mainly relevant for CDB minimum minetest version)

Reply:
yup, and textures in dirs is 5.4 only